backend/src/app.ts

```diff
@@ -1,3 +1,4 @@
+import 'middleware/auth';
```

Comment:
Instead of importing a whole script, I think it would be better to configure passport strategies in "middleware/auth" than either export strategies from there and configure them in this file or set config function there and call it in this file.